toolkit/tools/internal/logger/log.go

```diff
@@ -245,3 +246,29 @@ func setHookLogLevel(hook *writerHook, level string) (err error) {
 	hook.SetLevel(logLevel)
 	return
 }
+
+// FormatWarningBox formats a message into a box with a border. The box is automatically sized to fit the longest line.
+// Each line will be centered in the box.
+func FormatWarningBox(message []string) []string {
+	var (
+		corners    = []string{"╔", "╗", "╚", "╝"}
+		horizontal = "═"
+		vertical   = "║"
+	)
+	maxLineLength := 0
+	for _, line := range message {
+		if len(line) > maxLineLength {
+			maxLineLength = len(line)
+		}
+	}
+	// Count: 2 for corners, 2 for padding, longest string
+	lines := []string{corners[0] + strings.Repeat(horizontal, maxLineLength+2) + corners[1]}
```

Comment:
Seeing how `corners[X]` is used once for each `X`, I think we're not gaining much by keeping a separate `corners` slice, since the index `X` doesn't really say much to the reader. For clarity, I'd suggest just breaking it out to separate well-named constants like `const upperLeftCorner = "╔"`.

Reply:
Wait, I've got another idea. I'll need a few minutes but I'm inspired by padding and [this article](https://stackoverflow.com/questions/41133006/how-to-fmt-printprint-this-on-the-center).

Reply:
Split into 		cornerTL   = "╔", cornerTR   = "╗", cornerBL   = "╚", cornerBR   = "╝".